src/deploy/durable_objects.rs

```diff
@@ -1,65 +1,340 @@
-use crate::http;
+use std::{cell::RefCell, collections::HashMap};
+
 use crate::settings::global_user::GlobalUser;
+use crate::settings::toml::DurableObjects;
+use crate::{http, settings::toml::Target};
+
+use reqwest::blocking::Client;
+use serde::{Deserialize, Serialize};
+
+/*
+Durable Objects have a fun chicken and egg problem that makes them rather tricky to deploy,
+in the case of scripts that both implement and bind to a namespace... you must have a namespace
+id in order to create the binding to upload the script, but you need the script to create the
+namespace. What we did to get around this, was allow creating a namespace without a script,
+that just returns an error when you try to access it.
+
+When you use/implement a DO in the same script, we'll first initialize the namespace (if it does
+not exist already) with no script/class so we have an ID to use for the binding. After the script is
+uploaded, the namespace is finalized with the script and class filled in.
 
-use serde::Serialize;
+Scripts using durable objects should be written to handle errors accessing the object to handle
```

Comment:
Scripts should be written to handle problems talking to their durable objects anyways, so maybe this note is better off being left out...